src/CMakeLists.txt

```diff
@@ -185,7 +185,7 @@ if(CCTAG_WITH_CUDA)
   endif()
 
   target_compile_definitions(CCTag
-                         PUBLIC -DCCTAG_WITH_CUDA
+                         PUBLIC -DCCTAG_WITH_CUDA -DBOOST_NO_CXX11_CONSTEXPR
```

Comment:
maybe this can be removed now that we switched to c++14?

Reply:
works well locally, pushed to see on other platforms